cmd/antrea-agent/agent.go

```diff
@@ -607,6 +605,7 @@ func run(o *Options) error {
 		ovsBridgeClient,
 		proxier,
 		networkPolicyController,
+		mcastController,
```

Comment:
there is no point to inject this to agentQuerier, it could be passed to `installHandlers` directly like AgentNetworkPolicyInfoQuerier and ServiceExternalIPStatusQuerier

Reply:
updated